src/main/java/com/fvv/bookstore/controller/BookControllerImpl.java

```diff
@@ -54,9 +57,12 @@ public boolean addBook(final Book book) throws ControllerException {
 	/**
 	 * {@inheritDoc}
 	 */
-	public boolean updateBook(final Book book) throws ControllerException {
+	public boolean updateBook(final Book book) throws BookValidationException, ControllerException {
 		try {
+			this.validateBook(book);
 			return this.bookDao.updateBook(book);
+		} catch (BookValidationException e) {
```

Comment:
you can simply remove this catch here since you're re-throwing an existent one!

Reply:
ok